src/persistentReducedStream.spec.ts

```diff
@@ -277,3 +278,32 @@ test(
     m.expect(state$).toBeObservable(expected$);
   })
 );
+
+test(
+  'persistentReducedStream forwards namespace to reducers',
+  marbles((m) => {
+    const action$ = m.hot('  --n-', actions);
+    const expected$ = m.hot('1-2-', numbers);
+    const initialState = 1;
+
+    const verifyNamespaceReducer = reducer(
+      actionCreators.incrementOne,
+      (state, _, namespace) => {
```

Comment:
In the action, the namespace is within the `meta` object. This can be done for the reducer as well, or what do you think? It might make it easier to extend this in the future if more variables are passed down (Not sure if that is planned tho). I'm fine with this if you think this is the last 👍

Reply:
If there will be more then I think we need to start to name the arguments.

Reply:
I don't plan on passing along more variables currently. As long as we are in `0.x.x`, and are unafraid of breaking changes and refactoring, I think we should keep it like this for now, but it might make sense at some point to pass the whole action or meta object.